src/commands/install.ts

```diff
@@ -384,8 +384,12 @@ export async function install(
         warning.plugin === 'snowpack:rollup-plugin-catch-unresolved'
       ) {
         // Display posix-style on all environments, mainly to help with CI :)
-        const fileName = warning.id!.replace(cwd + path.sep, '').replace(/\\/g, '/');
-        logError(`${fileName}\n   ${warning.message}`);
+        if (warning.id) {
+          const fileName = warning.id.replace(cwd + path.sep, '').replace(/\\/g, '/');
```

Comment:
This `warning.id!` strong assertion was the problem (it doesn’t always exist).

Reply:
Ah! of course, great catch